src/TDebug.cpp

```diff
@@ -205,7 +242,26 @@ void TDebug::changeHostName(const Host* pHost, const QString& newName)
 
 /* static */ void TDebug::removeHost(Host* pHost, const QString hostName)
 {
-    auto identifier = TDebug::smIdentifierMap.take(pHost);
```

Comment:
This method (`TDebug::removeHost(...)`) **is expecting** to receive a `Host*` that is NOT valid - i.e. is a dangling pointer (not a `nullptr` but not valid/safe to dereference either) and it is **only** used to pull the matching entry from the lookup table and **never** to be dereferenced. So I think you do not need to touch this!

Reply:
Changes were made in response to crashes.